geoopt/manifolds/euclidean.py

```diff
@@ -52,3 +52,6 @@ def _logmap(self, x, y):
 
     def _dist(self, x, y):
         return (x - y).abs()
+
+    def rand_(self, x):
```

Comment:
overrides public method and that's why we loose pretty docstrings in jupyter, should override `_rand_` instead?